XbSymbolDatabase.c

```diff
@@ -971,15 +971,12 @@ bool XbSymbolScan(void* xbeData, xb_symbol_register_t register_func)
         bool bDSoundLibHeader = false;
 
         // Verify if title do contain DirectSound library section.
-        for (unsigned int v = 0; v < pXbeHeader->dwSections; v++) {
+        for (uint32_t v = 0; v < pXbeHeader->dwSections; v++) {
```

Comment:
should stick to `unsigned int` here. It's guaranteed to be large enough but gives the compiler more options to work with.

Reply:
Okay, I'm usually habit with size bound to avoid screwing up the uneven size problem in the past.